analyses/cell-type-nonETP-ALL-03/Azimuth_BM_level1.csv

```diff
@@ -0,0 +1,16 @@
+tissueType,cellName,geneSymbolmore1,geneSymbolmore2,shortName
```

Comment:
Thank you for updating this file to one that we can more easily track changes to! 

I wonder if it makes more sense to have a separate column for the cell ontology? I am not sure if that would upset ScType, but I suspect it will not matter, or we can remove it before passing it along. 

As to some of the ambiguous cell ontology values, I think it probably makes the most sense to just use whatever was provided by Azimuth as the best equivalent. In the case of early and late erythrocytes this will just mean that they have the same CL value: [`CL_0000764`](http://purl.obolibrary.org/obo/CL_0000764), and for "Other T" that would just be the generic T cell: [`CL_0000084`](http://purl.obolibrary.org/obo/CL_0000084).